website/blog/2023-01-12-version-071.md

```diff
@@ -0,0 +1,246 @@
+- **Write less C++ code**: you can now enable the New Architecture without having to add any C++ code in your app and the CLI app template has been cleaned of all the C++ code and the CMake files. [_React more here_](https://github.com/reactwg/react-native-new-architecture/discussions/101).
+- **Better encapsulation of iOS app setup**: On iOS, we followed a similar approach to Android and encapsulated most of the logic to set up the New Architecture in the `RCTAppDelegate` class, which will simplify upgrades in the future with less manual breaking changes.
+- **Better dependency management on iOS**: For library maintainers, we've added a new `install_module_dependencies` function to call inside your package `podspec` which will install all the required dependencies for the New Architecture.
+- **Bug fixes and better IDE support**: we fixed several bugs and issues (like [_better IDE support for Android_](https://github.com/reactwg/react-native-new-architecture/discussions/102)) that were reported by our users in the [_New Architecture Working Group_](https://github.com/reactwg/react-native-new-architecture/discussions).
+
+As a reminder, the New Architecture is still an experimental API experience as we iterate on changes to make adoption easier. Please try out the new simplified steps in the [_New Architecture documentation_](https://reactnative.dev/docs/next/new-architecture-intro) and post any feedback you have to the [_New Architecture Working Group_](https://github.com/reactwg/react-native-new-architecture/discussions).
+
+### Other Notable Fixes
+
+- **Better stack frame collapsing**: We've [updated the list of internal frames](https://github.com/react-native-community/cli/pull/1699) for React Native so LogBox will show your code more often rather than internal React Native frames, helping you to debug issues faster.
+- **Build time improvements:** We migrated assets to Maven Central for prefabs to improve build times (both iOS and Android) for Hermes in both the current and new architectures.
+- **Android template improvements**: The Android template was heavily cleaned and now fully relies on the React Native Gradle Plugin. You can find the configuration instructions directly inside the template or in the [_new dedicated page on the website_](https://reactnative.dev/docs/next/react-native-gradle-plugin).
+
+## Breaking changes
+
+- **Changes to Console Logging:** `LogBox.ignoreLog` no longer filters console logs. This means you will start seeing logs in the console that you have silenced in LogBox. See [this comment](https://github.com/facebook/react-native/pull/34476#issuecomment-1240667794) for more details.
+- **Removed AsyncStorage and MaskedViewIOS**: These components have been deprecated since version (TODO), so it’s time we remove them entirely. For alternatives, please check [_React Native Directory_](https://reactnative.directory/) for community packages that cover those use cases.
```

Comment:
This todo needs to be filled in. Anyone know the version number to add?

Reply:
0.59: https://github.com/facebook/react-native/blob/main/CHANGELOG.md#deprecated-8

Reply:
```suggestion
- **Removed AsyncStorage and MaskedViewIOS**: These components have been deprecated since version [0.59](https://github.com/facebook/react-native/blob/main/CHANGELOG.md#deprecated-8), so it’s time we remove them entirely. For alternatives, please check [_React Native Directory_](https://reactnative.directory/) for community packages that cover those use cases.
```